web/src/features/panels/ranking-panel/getRankingPanelData.ts

```diff
@@ -58,7 +58,7 @@ export const getRankedState = (
     })
     .filter(
       (zone) =>
-        zone.co2intensity !== undefined &&
+        zone.co2intensity !== undefined && zone.co2intensity !== null &&
```

Comment:
```suggestion
        Boolean(zone.co2intensity) &&
```
I would simplify this check to this, not only does it produce less code but it makes the code easier to understand IMO.

Reply:
Thanks for reply @VIKTORVAV99!
I like your solution, looks well too.
I have a question about this phrase: "But if you do you need to set up poetry and use that instead of pip." - can you be more explicit about it?
I've tryied to run poetry format but somehow I have this error..maybe someone can help about it..I would like to fix it and contribute to future issues since project is impressive!
"poetry run format
Traceback (most recent call last):
  File "/home/beast/.local/bin/poetry", line 5, in <module>
    from poetry.console.application import main
  File "/usr/lib/python3/dist-packages/poetry/console/__init__.py", line 1, in <module>
    from .application import Application
  File "/usr/lib/python3/dist-packages/poetry/console/application.py", line 3, in <module>
    from cleo import Application as BaseApplication
ImportError: cannot import name 'Application' from 'cleo' (/home/beast/.local/lib/python3.10/site-packages/cleo/__init__.py)"

Reply:
Have you followed the steps we have described here https://github.com/electricitymaps/electricitymaps-contrib/wiki/Set-up-local-environment?

Those should contain everything you need and if they don't please let me know or open a issue so we can update it.

Reply:
I've followed the guide and have troubles installing poetry...:
"
python3 --version      
Python 3.10.12
beast@DESKTOP-TFSG0II:/mnt/c/Users/admin/Documents/code js/electricity_fork$ pipx install poetry
⚠️  File exists at /home/beast/.local/bin/poetry and points to /home/beast/.local/bin/poetry, not
    /home/beast/.local/pipx/venvs/poetry/bin/poetry. Not modifying.
  installed package poetry 1.8.2, installed using Python 3.10.12
    - poetry (symlink missing or pointing to unexpected location)
done! ✨ 🌟 ✨
beast@DESKTOP-TFSG0II:/mnt/c/Users/admin/Documents/code js/electricity_fork$ poetry --version
Traceback (most recent call last):
  File "/home/beast/.local/bin/poetry", line 5, in <module>
    from poetry.console.application import main
  File "/usr/lib/python3/dist-packages/poetry/console/__init__.py", line 1, in <module>
    from .application import Application
  File "/usr/lib/python3/dist-packages/poetry/console/application.py", line 3, in <module>
    from cleo import Application as BaseApplication
ImportError: cannot import name 'Application' from 'cleo' (/home/beast/.local/lib/python3.10/site-packages/cleo/__init__.py)
"
I don't understand where is the issue and where it comes from